src/clr-angular/data/datagrid/datagrid-detail-body.spec.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { Component } from '@angular/core';
+
+import { TestContext } from './helpers.spec';
+import { ClrDatagridDetailBody } from './datagrid-detail-body';
+
+const content = 'Detail Pane';
+
+export default function(): void {
+  describe('ClrDatagridDetailBody component', function() {
+    describe('View', function() {
+      let context: TestContext<ClrDatagridDetailBody, FullTest>;
+
+      beforeEach(function() {
+        context = this.create(ClrDatagridDetailBody, FullTest);
+        context.detectChanges();
+      });
+
+      it('should wire up host bindings', () => {
+        expect(context.clarityElement.className).toContain('datagrid-detail-body');
+      });
+
+      it('projects content into the detail pane ', () => {
```

Comment:
nit: isn't this more like a Typescript API test? I see that we have it in both places throughout the library though.

Reply:
I think the grouping of tests is difficult in some cases, and I'd rather not create multiple groups for something this simple. Either way, its the view ensuring we basically have ng-content right?

/shrug